weave/trace_server/clickhouse_trace_server_batched.py

```diff
@@ -537,12 +537,11 @@ def ops_query(self, req: tsi.OpQueryReq) -> tsi.OpQueryRes:
                 conds.append("object_id IN {op_names: Array(String)}")
                 parameters["op_names"] = req.filter.op_names
 
-            if req.filter.latest_only:
-                conds.append("is_latest = 1")
-
         ch_objs = self._select_objs_query(
             req.project_id,
             conditions=conds,
+            parameters=parameters,
```

Comment:
why was this never passed in??

Reply:
ops_query is not used :( so it likely has a few bugs